### ISSUE TITLE
file path in ios

### ISSUE THREAD

**Issue by sotos89** [j3k0/cordova-plugin-openwith#46]
when i want to upload a photo in ios from share as i  can see i don't have the photo file path. The only think i have that look like path is extension://index=0 but it does not work while i am trying to upload photo.

**Comment by optimusprime13** [comment]
From iOS 12 onwards, the file content is obtained on simulator. Did you test on device ?

**Comment by sotos89** [comment]
yes i tested on simulator and device as well, because of i am using filetransfer plugin to upload fie and it doesn't support base64, the workaround i found is from base64 to create a blob file, copy it in a temporary file and then I could upload it. Its not the best way but it's a solution and it works.
